pkg/skaffold/runner/new.go

```diff
@@ -233,6 +236,107 @@ func getSyncer(cfg sync.Config) sync.Syncer {
 	return sync.NewSyncer(cfg)
 }
 
+/*
+The "default deployer" is used in `skaffold apply`, which uses a `kubectl` deployer to actuate resources
```

Comment:
can we move this to `pkg/skaffold/apply/deployer.go` or something similar?

Reply:
yeah it probably should....i didn't do it here because it's a bigger refactor and i didn't want to mix organization patterns. let me go take a stab at the refactor though, new PR pending

Reply:
my comment was only scoped to the new function added. Are you talking about refactoring your other PR? This function doesn't seem to be used anywhere else.

Reply:
ahh sorry i misread your comment, i thought you were suggesting putting it in `pkg/skaffold/deploy`. i actually think that's where it should be, so the runner would make calls like `deploy.GetDeployer` and `build.GetBuilder` instead of having private `getDeployer` and `getBuilder` functions which do pre-processing and then instantiate.

IMO this pattern makes more sense, but is a bigger refactor, so rather than set an example with this PR (but have mismatched organization patterns) i decided to just put it alongside the pre-existing `getDeployer` function. i can move it to an `apply` package so it's more clear what it's used for though.

separately - this PR just adds this logic for ease of review, it's only actually used in the other PR which adds the `apply` command.

Reply:
make sense. I thought it's putting too many things in `new.go`.